### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Cryptolilly** [vercel/next.js#55673] [Show and tell]
Is it necessary to have knowledge on JavaScript before getting to Next.js

**Comment by icyJoseph** [comment]
Somewhat yes. 

The Next.js learn docs have this section:

- https://nextjs.org/learn/foundations/from-javascript-to-react/essential-javascript-react

Which is a great starting point. 

You'd also benefit from understanding Browser APIs and Node.js APIs
